runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldSwitcherPanel.java

```diff
@@ -247,6 +266,67 @@ void populate(List<World> worlds)
 					break;
 			}
 
+			if (exclusiveTypeFilter)
+			{
+				if (!world.getTypes().contains(WorldType.PVP) &&

```

Comment:
Our code convention for multiline boolean operations are to put the operators at the start of continued lines like so:

```java
a
	&& b
	&& c;
```

Reply:
Additionally, this check for `world.getTypes()` containing at least one of the many types here can be implemented more cleanly using [`Collections.disjoint(A, B)`](https://docs.oracle.com/javase/8/docs/api/java/util/Collections.html#disjoint-java.util.Collection-java.util.Collection-)